tools/utils/utils.go

```diff
@@ -0,0 +1,77 @@
+package utils
+
+import (
+	"io/ioutil"
+
+	"github.com/ghodss/yaml"
+	"github.com/odo-devfiles/registry/tools/types"
+)
+
+const (
+	runGroup = "run"
+)
+
+// IsDevfileSupported checks if devfile v2 is supported
+func IsDevfileSupported(devfile types.Devfile) bool {
+
+	hasComponentContainer := false
+	hasComponentContainerName := false
+	hasRunGroupCommand := false
+
+	for _, component := range devfile.Components {
```

Comment:
Could we simplify these two for loops a bit? Maybe something like:

```
hasSupportedContainer := false
hasRunGroupCommand := false
for _, component := range devfile.Components {
    if component.Container != nil && component.Container.Name != "" {
        hasSupportedContainer = true
        break
    }
}

for _, command := range devfile.Commands {
    if hasRunGroupCommand = command.Exec != nil && command.Exec.Group != nil && command.Exec.Group.Kind == runGroup {
       hasRunGroupCommand = true
       break
    }
}

return hasSupportedContainer && hasRunGroupCommand
```

Reply:
And maybe it makes sense to separate out each for loop into a function?

Reply:
Updated with feedback, I had just copy pasted it from odo repo before. It looks cleaner now.